### PR TITLE
old minetest 5.0.1

### PR DESCRIPTION
**Pull request by johnlockard**
Updated source and resource to current version.
Removed unnecessary X11 dependency.
Added simple Test Block.

To do: Figure out how to update the Bottle Block.
To do: Figure out how to run unit tests in the Test Block.

- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by SMillerDev** [comment]
`C: 67:  1: Trailing whitespace detected. `

File: Formula/minetest.rb [hunk 1]
```diff
@@ -65,4 +64,14 @@ def caveats
       "/Applications/minetest.app/Contents/MacOS/minetest --server".
     EOS
   end
+    
+  test do
+    #
+    # Commands --help, --version will produce a readable output without creating a file.
+    #
+    # Commands --info, --run-unittests, --speedtests, --trace, and
+    # --videomodes require creating user data directory for output.
```
Comment (fxcoudert) [inline Formula/minetest.rb]: We don't really mind creating a user datadir. In our test sandbox, the datadir will be erased after the test is run anyway.

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/minetest.rb (hunk 1 above)
Comment (johnlockard) [reply Formula/minetest.rb]: Okay.

I'll have to do a bit of reading in the documentation on how to do that. Maybe, by the next update I'll have figured it out.

Comment (johnlockard) [reply Formula/minetest.rb]: I've changed the file and got four more modes working.

Comment (johnlockard) [reply Formula/minetest.rb]: Scratch that. Irrlicht Engine reporting No doublebuffering available.
That leaves only one option.

**Comment by SMillerDev** [comment]
Could you squash all your commits and make the name `minetest 5.0.1`

**Comment by fxcoudert** [comment]
We do not want to delete minetest.

**Pull request closed without merging**